com.mudeprolinux.whakarere.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.mudeprolinux.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+
+finish-args:
+  # X11 and IPC access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # PulseAudio
+  - --socket=pulseaudio
+
+modules:
+  - name: Whakarere
+    buildsystem: simple
+    build-commands:
+      - install -D whakarere.py /app/bin/whakarere
+      - install -Dm644 whakarere.png /app/share/icons/hicolor/512x512/apps/com.mudeprolinux.whakarere.png
+      - install -Dm644 whakarere.desktop /app/share/applications/com.mudeprolinux.whakarere.desktop
+      - install -Dm644 com.mudeprolinux.whakarere.appstream.xml /app/share/appdata/com.mudeprolinux.whakarere.appstream.xml
```

Comment:
this is the wrong file name.

Reply:
I just figured it out that I didn´t make the change on the manifest.